test/e2e_node/device_plugin_test.go

```diff
@@ -227,21 +227,13 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 			pod1, err := e2epod.NewPodClient(f).Get(ctx, pod1.Name, metav1.GetOptions{})
 			framework.ExpectNoError(err)
 
-			ensurePodContainerRestart(ctx, f, pod1.Name, pod1.Name)
-
-			ginkgo.By("Confirming that device assignment persists even after container restart")
-			devIDAfterRestart := parseLog(ctx, f, pod1.Name, pod1.Name, deviceIDRE)
-			framework.ExpectEqual(devIDAfterRestart, devID1)
-
-			ginkgo.By("Restarting Kubelet")
-			restartKubelet(true)
-
 			ginkgo.By("Wait for node to be ready again")
 			e2enode.WaitForAllNodesSchedulable(ctx, f.ClientSet, 5*time.Minute)
```

Comment:
without kubelet restart, these two lines seem redundant, surely the `gingko.By`. We may want to keep the `waitForallNodesSchedulable` for extra safety though